dll/cpl/desk/lang/pt-PT.rc

```diff
@@ -36,26 +36,26 @@ BEGIN
     PUSHBUTTON "&Personalizar...", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
 END
 
-IDD_SCREENSAVER DIALOGEX 0, 0, 246, 216
+IDD_SCREENSAVER DIALOGEX 0, 0, 246, 202
```

Comment:
That size was already used in pt-BR ? What about the other languages?

Reply:
**It is synced back** to en-US dimensions with my changes, read the commit changelog. I explained everything in detail. Other translators behforehand changed the entire dlgs dimensions to differ from en-US, which is not desireable. Especially not within a property-sheet where several sheets are placed within.